### ISSUE TITLE
beginner troubles to view

### ISSUE THREAD

**Issue by michaelaye** [dask/dask-labextension#113]
Even so @mrocklin 's Dask delayed video

https://www.youtube.com/watch?v=SHqFmynRxVU

does not show it in the notebook, I had to do the following to be able to make the dashboard items of the lab extension work:

* Launch a LocalCluster (expected)
* Not expected: Inside notebook, create a client object, connecting to the established LocalCluster using:

```python
from dask.distributed import Client

client = Client("tcp://127.0.0.1:45906")
client
```

Without that, delayed execution still worked (even so I didn't create a client), but wasn't shown in the dashboard items I opened using the icons of the dask-labextension.
I guess, delayed creates a new cluster (and client) on its own, when no client object is present?

After connecting the client as above, the dashboard items worked.

Maybe a useful addition to the ReadMe?

**Comment by michaelaye** [comment]
Addition: Above works for a remote jupyterlab running on Linux, using Safari on macOS.
I still cannot make the dashboard work for a LocalCluster running on macOS.
Connecting to the LocalCluster started with the lab extensions works and calculations work as well,
but when trying to open a dashboard item, I get this (in between the UserWarnings I reported in #112 ):

```python
/Users/klay6683/miniconda3/envs/py37/lib/python3.7/site-packages/tornado/web.py:1699: RuntimeWarning: coroutine 'WebSocketHandler.get' was never awaited
  result = await result
```

Maybe related?

**Comment by mrocklin** [comment]
I think that you might want to look through Dask's documentation on schedulers.  If you don't have a dask.distributed client active then Dask uses a local thread pool or processing pool.  This is well documented here: https://docs.dask.org/en/latest/scheduling.html

Closing.  Thanks for raising the issue @michaelaye .  If you still think that there is some problem then please speak up and we'll reopen.

**Comment by michaelaye** [comment]
If https://docs.dask.org/en/latest/scheduling.html is required reading before using the dask-labextension, I guess it could be mentioned in the ReadMe?
Although, pointing the user to injecting the Client reference code into the lab session might just be sufficient.

**Comment by mrocklin** [comment]
Understanding how Dask works generally is generally useful, yes.  I'm not
sure that that page should be called out specifically though, there are
lots of things that one should know and we probably can't reference all of
them everywhere and still keep things readable.

**Comment by michaelaye** [comment]
For the future reader: dask_labextension 1.1 actually does link the dashboard automatically, so after launching a LocalCluster with the extension, one doesn't need to connect to that client within a notebook anymore and the dashboard items are working.
Also the aforementioned Mac-only issue is solved with version 1.1.
Note that I was doing a fresh conda-forge install but something in my (admittedly large) environment was having dependency constraints in the way that I only got version 0.3.3 from conda-forge. "Brutally" updating with pip "solved" that issue. (See #112  also).
